ui/shared/utils/constants.js

```diff
@@ -809,6 +810,15 @@ const VARIANT_SORT_OPTONS = [
       Object.keys(a.transcripts || {}).reduce(
         (acc, geneId) => (genesById[geneId] ? acc + genesById[geneId].omimPhenotypes.length : acc), 0),
   },
+  {
+    value: SORT_BY_TAGGED_DATE,
+    text: 'Last Tagged',
+    comparator: (a, b, genesById, user, tagsByGuid) =>
+      (b.tagGuids.map(
+        tagGuid => (tagsByGuid[tagGuid] || {}).lastModifiedDate).sort()[b.tagGuids.length - 1] || '').localeCompare(
+        a.tagGuids.map(tagGuid => (tagsByGuid[tagGuid] || {}).lastModifiedDate).sort()[a.tagGuids.length - 1] || '',
+      ),
```

Comment:
The last parenthesis matches the one of `localeCompare(` in line 818, not the one in line 817. Rearrange the alignment of parentheses in these lines will make them more readable.

Reply:
The indentation is correct, if I change it  I get a JS linting error so I am leaving this as is